### PR TITLE
Linkedin connection requests issue#278

### LINKED ISSUE #278: Send LinkedIn Connection Requests
# Description

Create a Script to send LinkedIn Connection Requests in bulk from this [endpoint](https://www.linkedin.com/mynetwork/) with an argument as a Custom Connection Message. You can specify how many people you may want to send the Connection Request. 

## Language

- [X] Bash
- [X] Go
- [X] Javascript
- [X] Python

## Type of issue

_Please delete options that are not relevant._

- [X] Feature

### PR DESCRIPTION
**Pull request by SANKET7738**
# Description

A script in javascript to send a bulk connection request.

Fixes #278 

## Type of change
- Updated README
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: JavaScript/Bulk_LinkedIn_requests/README.md [hunk 1]
```diff
@@ -2,3 +2,16 @@
 
 Scripts in JavaScript to send bulk connection requests
 
+1. Open your browser and login into your Linkedin account.
+2. Navigate to ```linkedin.com/search/results/people/```.
+3. Right click then click on **Inspect** or use ```ctrl+shift+i```.
+4. Open console as shown below.
+
+![endpoint](https://github.com/SANKET7738/random/blob/master/bulk_linkedin_requests/demo-imgs/endpoint.png)
```
Comment (HarshCasper) [inline JavaScript/Bulk_LinkedIn_requests/README.md:10]: You can make use of something like `Imgur` to refer these endpoints.

File: JavaScript/Bulk_LinkedIn_requests/README.md [hunk 2]
```diff
@@ -2,3 +2,16 @@
...
+
+5. Open the script.
+6. You can change the message according to you at line 56.
+7. Now copy the whole code and paste it into the console section.
+8. Wait for the script to do its job.
+
+![demo](https://github.com/SANKET7738/random/blob/master/bulk_linkedin_requests/demo-imgs/demo.png)
```
Comment (HarshCasper) [inline JavaScript/Bulk_LinkedIn_requests/README.md:17]: You can make use of something like `Imgur` to refer these endpoints.

**Review by HarshCasper — COMMENTED** [review]
Kindly make these changes before the final merge.

**Comment by HarshCasper** [comment]
@SANKET7738 Are you working on the same?

**Comment by SANKET7738** [comment]
So sorry I forgot about this task. I have made the changes as you requested I will put a new PR for the same.

**Pull request closed without merging**